sdks/java/core/src/main/java/org/apache/beam/sdk/util/state/InMemoryTimerInternals.java

```diff
@@ -144,27 +150,26 @@ public String toString() {
     return MoreObjects.toStringHelper(getClass())
         .add("watermarkTimers", watermarkTimers)
         .add("processingTimers", processingTimers)
+        .add("synchronizedProcessingTimers", processingTimers)
         .add("inputWatermarkTime", inputWatermarkTime)
         .add("outputWatermarkTime", outputWatermarkTime)
         .add("processingTime", processingTime)
         .toString();
   }
 
-  /** Advances input watermark to the given value and fires event-time timers accordingly. */
-  public void advanceInputWatermark(
-      TimerCallback timerCallback, Instant newInputWatermark) throws Exception {
+  /** Advances input watermark to the given value. */
+  public void advanceInputWatermark(Instant newInputWatermark) throws Exception {
     checkNotNull(newInputWatermark);
     checkState(
         !newInputWatermark.isBefore(inputWatermarkTime),
         "Cannot move input watermark time backwards from %s to %s",
         inputWatermarkTime,
         newInputWatermark);
     WindowTracing.trace(
-        "TestTimerInternals.advanceInputWatermark: from {} to {}",
+        "InMemoryTimerInternals.advanceInputWatermark: from {} to {}",
```

Comment:
Awkward that this message made it in to the codebase. Can we use `getClass().getSimpleName()`?